src/pages/EntityDetails/EntityDetails.tsx

```diff
@@ -212,6 +224,29 @@ const EntityDetails = () => {
           </div>
         </div>
       </Header>
+      {modelWatcherError ? (
+        <Strip className="u-no-padding--bottom" shallow>
+          <Notification
+            className="u-no-margin--bottom"
+            severity="negative"
+            title="Error"
+          >
+            {modelWatcherError === "timeout" ? (
+              <span>{Label.MODEL_WATCHER_TIMEOUT}</span>
+            ) : (
+              <span>{Label.MODEL_WATCHER_ERROR}</span>
+            )}
+            {typeof modelWatcherError === "string" &&
```

Comment:
If the type changes to a string this could become:

```suggestion
            {modelWatcherError &&
```

Reply:
Done.